src/test/sstruct.c

```diff
@@ -1368,6 +1368,20 @@ DistributeData( ProblemData   global_data,
                                         pdata.graph_iuppers[box],
                                         ilower, iupper,
                                         int_ilower, int_iupper);
+
+                  /* Correct intersected box extents */
```

Comment:
Why is this different from the code on master?  Is there something wrong on master as well?